atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/DbkvsPostgresKeyValueServiceTest.java

```diff
@@ -38,28 +39,18 @@ protected KeyValueService getKeyValueService() {
         return kvs;
     }
 
-    @Test
-    public void dontThrowWhenCreatingTheSameLongTable() throws Exception {
-        TableReference longTableName = TableReference.create(TEST_NAMESPACE, TEST_LONG_TABLE_NAME);
-
-        keyValueService.createTable(longTableName, AtlasDbConstants.GENERIC_TABLE_METADATA);
-        keyValueService.createTable(longTableName, AtlasDbConstants.GENERIC_TABLE_METADATA);
-
-        keyValueService.dropTable(longTableName);
-    }
-
     @Test(expected = RuntimeException.class)
     public void throwWhenCreatingDifferentLongTablesWithSameFirstCharactersUntilTheTableNameLimit() {
         String tableNameForFirstSixtyCharactersToBeSame = StringUtils.left(TEST_LONG_TABLE_NAME,
-                AtlasDbConstants.ATLASDB_POSTGRES_TABLE_NAME_LIMIT - TEST_NAMESPACE.getName().length()
+                PostgresDdlTable.ATLASDB_POSTGRES_TABLE_NAME_LIMIT - TEST_NAMESPACE.getName().length()
```

Comment:
These tests could also be moved to AbstractDbKvs**Test, if that class gains an abstract method to get the table name length limit (assuming all implementations of AbstractDbKvs are for a KVS that has such a limit).